### PR TITLE
feat(docs): Add feedback section

### PR DESCRIPTION
**Pull request by IsAmrish**
Signed-off-by: isamrish <askmaurya48@gmail.com>

This commit will add a feedback section at the bottom of every page

Feedback section contains two buttons - yes and no. When a user clicks on
any button, a relevant event info will be sent to GA.

### CODE REVIEW

File: website/node_modules/docusaurus/lib/core/Feedback.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class Feedback extends React.Component {
+    render() {  
+     return(
+       <div>
+        <h2>Feedback:</h2>
+        <hr/>
+        <p id="feedback-question">Was this page helpful?</p>
+        <button className="edit-page-link button feedback-button" id="feedback-yes">Yes</button>
+        <button className="edit-page-link button feedback-button" id="feedback-no">No</button>
+        <p id="feedback-response" className="feedback-response__hidden">Thanks for the feedback. If you have a specific, answerable question about how to use OpenEBS, please join <a href="https://slack.openebs.io" target="_blank" rel="noopener">OpenEBS community</a></p>
```
Comment (kmova) [inline website/node_modules/docusaurus/lib/core/Feedback.js:14]: Suggested content. 

```
<p id="feedback-response" className="feedback-response__hidden">Thanks for the feedback. Open an issue in the <a href="https://github.com/openebs/openebs/issues/new" target="_blank" rel="noopener">GitHub repo</a> if you want to report a problem or suggest an improvement. Engage and get additional help on <a href="https://github.com/openebs/openebs/issues/new" target="_blank" rel="noopener">https://kubernetes.slack.com/messages/openebs/</a></p>
```

File: website/node_modules/docusaurus/lib/core/Feedback.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// This file is not part of core lib files
+
+const React = require('react');
+
+class Feedback extends React.Component {
+    render() {  
+     return(
+       <div>
+        <h2>Feedback:</h2>
```
Comment (kmova) [inline website/node_modules/docusaurus/lib/core/Feedback.js:9]: Remove `:`

**Comment by sagarkrsd** [comment]
Thanks for the PR @IsAmrish, should we make the `Feedback` button a little bold, maybe it will give a highlighted feeling to the user.

File: website/node_modules/docusaurus/lib/core/Site.js [hunk 3]
```diff
@@ -185,6 +185,57 @@ class Site extends React.Component {
                 __html: `(function(){ window.ldfdr = window.ldfdr || {}; (function(d, s, ss, fs){ fs = d.getElementsByTagName(s)[0]; function ce(src){ var cs = d.createElement(s); cs.async=1; cs.src = src; setTimeout(function(){fs.parentNode.insertBefore(cs,fs)}, 1); } ce(ss); })(document, 'script', 'https://lftracker.leadfeeder.com/lftracker_v1_bMwm7E2pzw4gOZyA.js'); })();`,
               }}
             />
+            <script async src="https://www.googletagmanager.com/gtag/js?id=UA-92076314-12"></script>
```
Comment (IsAmrish) [inline website/node_modules/docusaurus/lib/core/Site.js:188]: @kmova @sagarkrsd  -- added GA tracking code here which will load on every page!!

File: website/static/css/custom.css [hunk 4]
```diff
@@ -403,3 +403,23 @@ redoc .operation-type {
   background-color: #FEF0EB;
   border-radius: 15px;
 }
+
+/* Adding Custom CSS for Feedback section */
```
Comment (kmova) [inline website/static/css/custom.css:407]: Can you use the button color from openebs.io? Same size and colors as the "Copy" button at the bottom of the openebs.io page.

File: website/node_modules/docusaurus/lib/core/Feedback.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+class Feedback extends React.Component {
+    render() {  
+     return(
+       <div>
+        <h2>Feedback</h2>
+        <hr/>
+        <p id="feedback-question">Was this page helpful?</p>
+        <button className="feedback-button" id="feedback-yes">Yes</button>
+        <button className="feedback-button" id="feedback-no">No</button>
+        <p id="feedback-response" className="feedback-response__hidden">Thanks for the feedback. Open an issue in the <a href="https://github.com/openebs/openebs/issues/new" target="_blank" rel="noopener">GitHub repo</a> if you want to report a problem or suggest an improvement. Engage and get additional help on <a href="https://github.com/openebs/openebs/issues/new" target="_blank" rel="noopener">https://kubernetes.slack.com/messages/openebs/</a>.</p>
```
Comment (kmova) [inline website/node_modules/docusaurus/lib/core/Feedback.js:14]: Fix the link to the Slack channel.

**Pull request merged**